apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/PatientE2EBase.java

```diff
@@ -63,21 +76,21 @@ public void testReadWhenExistingPatientExpect200() {
   }
 
   /**
-   * Verifies that Patient searchByLogicalId returns a 200 and response when the beneficiary exists
-   * in the DB but has no {@link BeneficiaryHistory} or MedicareBeneficiaryIdHistory records.
-   * Primarily this checks that the table joins do not cause any issue retrieving the patient when
-   * there is nothing found in the history table.
+   * Verifies that Patient read returns a 200 and response when the beneficiary exists in the DB but
+   * has no {@link BeneficiaryHistory} or MedicareBeneficiaryIdHistory records. Primarily this
```

Comment:
MedicareBeneficiaryIdHistory is no more.

Reply:
Thanks! must have been a relic that wasnt removed when the table was destroyed